### ISSUE TITLE
other kinds of thing have 'callSign' beyond radio broadcasts, esp. vehicles/boats and non-radio broadcast

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#2372]
Currently,

* We have a sub-type of BroadcastService called RadioBroadcastService.
* callSign lists RadioBroadcastService as a type where it is expected.

https://schema.org/callSign

I propose we expand the domain to also anticipate BroadcastService and Vehicle

For examples see [Wikidata](https://w.wiki/AZi)

```
SELECT DISTINCT ?x ?tLabel WHERE { 
  ?x wdt:P2317 ?cs.
  ?x wdt:P31 ?t .
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en" }
}
ORDER BY ?tLabel
```

**Comment by danbri** [comment]
Text would need adapting, e.g. from "The official callsign for the radio broadcast." to "An official callsign for something, e.g. a radio broadcast, shipping boat etc."

**Comment by danbri** [comment]
This could also be documented as a sub-property of /identifier

**Comment by danbri** [comment]
```SELECT distinct ?t2Label  WHERE { 
  ?x wdt:P2317 ?cs.
  ?t wdt:P279 ?t2 .
  ?x wdt:P31 ?t .
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en" }
}
ORDER BY ?tLabel``` 

... shows more detail on the types in query.wikidata.org (could add [counts](https://www.wikidata.org/wiki/Wikidata:SPARQL_query_service/queries#Counting)...)

**Comment by mfhepp** [comment]
And please include person in the domain for callSign, since licensed amateur radio operators also have official callsigns (typically using the same naming scheme as e.g. US radio stations).

---------------------------------------
martin hepp
www:  http://www.heppnetz.de/
email: mhepp@computer.org

**Comment by danbri** [comment]
Good point @mfhepp - let's do that. Also @vholland provides an example of a non-radio broadcast service: "call signs are also used for television. So for example, WQED is the call sign for the PBS affiliate in Pittsburgh.".

**Comment by danbri** [comment]
Ignore the leaseLength commit tagged here, it was meant for #2373

**Comment by RichardWallis** [comment]
Implemented in commit 61784b5
